tensorflow/python/estimator/run_config.py

```diff
@@ -444,6 +451,7 @@ def __init__(self,
         `Operation` and returns the device string. If `None`, defaults to
         the device function returned by `tf.train.replica_device_setter`
         with round-robin strategy.
+      protocol: The protocol used when starting server.
```

Comment:
Can you mention this is optional field? And None means defaults to grpc?

Thanks

Reply:
Done.